src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,14 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <>
+    <article className={todo.completed
+      ? 'TodoInfo TodoInfo--completed'
+      : 'TodoInfo'
+    }
+    >
+      <h1 className="TodoInfo__title">{todo.title}</h1>
+      <UserInfo user={todo.user} />
```

Comment:
Let's imagine that the user is not exist 🤔 
![image](https://github.com/mate-academy/react_static-list-of-todos-js/assets/99403198/d807aef6-b186-46ee-a7bb-bbc8b91bac37)